MANIFEST.in

```diff
@@ -8,6 +8,9 @@ include *.md
 recursive-include flash *.md
 recursive-include flash *.py
 
+# Include assets
+recursive-include flash/assets *
```

Comment:
I may list what kind of images or file, `*` is quite a wild card...

Reply:
Done :)